src/slider/slider.scss

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-s+$iui-xxs;
+
+@mixin iui-slider-component-container {
+  display: inline-flex;
+
+  span.iui-left,
```

Comment:
Is there a reason span is used for css rules? It increases specificity which is not what we might want

Reply:
I used it specifically to increase specificity, as there are  other items that use `iui-left` that are decedents of `iui-slider-component-container` that I do not want this rule applied.

Reply:
hmm... Should we think about having a unique class instead? We already faced issues with the repeatance of the classes..

Reply:
What class name would you like? I could change it to something like "iui-slider-min" and "iui-slider-max" if you prefer.

Reply:
Yeah, I think that would suit better and reduce the specificity too.

Reply:
I renamed